confluent_server/bin/confluent_selfcheck

```diff
@@ -259,6 +260,15 @@ if __name__ == '__main__':
                     mac = rsp[key].get('value', None)
                     if mac:
                         macok = True
+            #adding new code to check if the response is something like net.<something>switch
+            for key in rsp:
+                if key.startswith('net.')and key.endswith('switch'):
+                    switch_value = rsp[key].get('value',None)
+                    if switch_value in valid_nodes:
+                        if args.node in valid_nodes:
```

Comment:
args.node shouldn't be checked at all in this case, args.node was checked elsewhere, we only check if switch_value is a valid node